po4a-stats.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# You need gettext
```

Comment:
You should check for it ;-)

Reply:
Since this script is called by another that already checks, I don't see the need. I shouldn't have included that reference here.

Reply:
Ok. Then just remove this comment.

Reply:
Done